tests/unit/QueryBuilder/ExportQueryBuilderTest.php

```diff
@@ -336,20 +336,20 @@ public function provideSuccessData(): Generator
             // @codingStandardsIgnoreStart
             <<<SQL
             SELECT `some_table`.`id`, `some_table`.`name`, `some_table`.`height`, `some_table`.`birth_at` FROM `some_schema`.`some_table` 
-            WHERE (`some_table`.`id` IN UNNEST(@dcValue1)) AND (SAFE_CAST(`some_table`.`id` AS INTEGER) NOT IN UNNEST(@dcValue2)) 
-            AND (SAFE_CAST(`some_table`.`height` AS NUMERIC) <> @dcValue3)
+            WHERE (`some_table`.`id` IN UNNEST(@dcValue1)) AND (`some_table`.`id` NOT IN UNNEST(@dcValue2)) 
+            AND (`some_table`.`height` <> @dcValue3)
             SQL,
             // @codingStandardsIgnoreEnd
             [
                 'dcValue1' => [
-                    'foo',
-                    'bar',
+                    "SAFE_CAST('foo' AS INT)",
+                    "SAFE_CAST('bar' AS INT)",
                 ],
                 'dcValue2' => [
-                    50,
-                    60,
+                    "SAFE_CAST('50' AS INT)",
+                    "SAFE_CAST('60' AS INT)",
                 ],
```

Comment:
:thinking: ty jo to funguje jako query parameters? Bych čekal že ti to ten SAFE_CAST nedovolí poslat jako parametr.

Reply:
no funguje jen castecne. ono to pak ale vyhodnoti jako string takze oescapuje. takze to tam budu muset poslat jako hodnotu. co bys rikal na https://github.com/keboola/php-storage-driver-bigquery/pull/126/commits/a4dfda3be28972b1c145a5850256880200033248 ? pripadne muzu udelat tu hodnotu v safe_cast jako param, takze by to bylo neco jako `SAFE_CAST(:dcValue AS NUMERIC)`

Reply:
Ta hodnota v safe_cast by byla víc safe :) šel bych spíš tím směrem, ono to pak ide jako parametry do BQ takže nemusíš řešit další kravince jako SQL injection.

Reply:
ok. zacinam mit pocit, jestli se nam ten SAFE_CAST vubec vyplati, ale ok, zkusim